helm/minio/values.yaml

```diff
@@ -316,21 +316,9 @@ policies: []
 #
 ## Additional Annotations for the Kubernetes Job makePolicyJob
 makePolicyJob:
-  podAnnotations: {}
-  annotations: {}
-  securityContext:
-    enabled: false
-    runAsUser: 1000
-    runAsGroup: 1000
-    fsGroup: 1000
```

Comment:
why are these removed?

Reply:
Because I have merged jobs into one,public job parameters are merged under postJob(line 418-424).

Reply:
So does that mean that it might break for users who might be using this ?